index.html

```diff
@@ -215,7 +233,14 @@ <h2 class="contact-me"><b>Contact me</b></h2>
         <button id="contact-btn" class="btn contact-btn" type="submit"> Get in touch</button>
         </form>
       </footer>
-  <script src="index.js" type="text/javascript"></script>
+
+      
+
+
+      <div id="overly" class="hidden"></div>
+
+      <div id="popupCard" class="popup-hidden"></div>
+  <script  defer src="index.js" type="text/javascript"></script>
```

Comment:
- [ ] as requested by the previous reviewer  consider moving the script tag into the <head> tag because the defer will allow your document to load before executing the script. If you want to leave it at the bottom of your page as you've done, you should remove the defer as the document will load first and then execute the script.
- [ ] same applies to this [code ](https://github.com/Jessekay/My-portfolio/blob/d03eaeeec29828a8ac4f2476f144ffbf8cccf6e4/index.html#L40)